fritzconnection/lib/fritzwlan.py

```diff
@@ -44,15 +46,14 @@ def get_beacon_security(instance, security):
     """
     if not security:
         info = instance.get_info()
```

Comment:
`security = NO_PASS` should get added here, so in case that the device does not support the `POSSIBLE_BEACON_TYPES_KEY` attribute `NO_PASS` is returned.